### DISCUSSION TITLE
wire:confirm not work with wire:submit in form

### DISCUSSION THREAD

**Discussion by ShekhSaifuddin007** [livewire/livewire#7138] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. In the blade file create a form,
2. In documentation says `Livewire makes this easy to do by adding wire:confirm in addition to any action (wire:click, wire:submit, etc.).`

```html
<form wire:submit="store">
    <div>
         <label>Name <span class="text-red-600">*</span></label>
         <input type="text" wire:model="name">

         <x-input-error for="name" />
     </div>

     <div class="flex justify-end mt-5 mb-2">
         <button
             type="submit"
             wire:confirm="Are you sure..?"
         >
            Create
       </button>
    </div>
</form>
```

It only works with wire:click.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by adriansio** [comment]
Encountered the same bug after watching Episode **Bulk Actions** from the Data Tables screencasts.

If you move the `wire:confirm` to the form, it requests confirmation but if you click `Cancel`, the form/button stay disabled.

**Reply by jdion84** [reply]
same here

**Reply by jakewhiteley** [reply]
Yup - this is being discussed in https://github.com/livewire/livewire/discussions/7176. I am just submitting a failed testcase to get this bug investigated.

The PR: https://github.com/livewire/livewire/pull/8039
